docs/reference/configuration.mdx

```diff
@@ -292,6 +299,74 @@ rate limiter. This could be combined with a second instance of
 
 </Note>
 
+## `ipv6Subnet`
```

Comment:
We'll need to rework this, maybe the configuration page gets level-2 headings for `rateLimit` and `ipKeyGenerator`, and then all of the options themselves become level-3 headers?

Alternatively we split them into two pages, but I don't really like that.